rules/php80/tests/Rector/Class_/ClassPropertyAssignToConstructorPromotionRector/Fixture/skip_variadic_arguments.php.inc

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace Rector\Php80\Tests\Rector\Class_\ClassPropertyAssignToConstructorPromotionRector\Fixture;
+
+class SkipVariadicArguments
+{
+    private string $foo;
+    /** @var string[] */
+    private array $bars;
+
+    public function __construct(
+        string $foo,
+        string ...$bars
```

Comment:
Skipping the variadic would be better here, so it's clear why this fixture was added.
The other property can be dropped

Reply:
I wanted to make sure that it will still handle properly supported promotion while skipping not supported one. But if you think that single, skipped argument will be good enough, I will remove `$foo`. Maybe it should be tested in other fixture (ensure promoting and skipping can be done in the same constructor).

Reply:
I understood that and used to write tests the same way. Recently I'm more into minimal-scope appraoch, with the minimal code possible that shows the bug. It makes reading fixtures from years hindsigh much easier than longer snippets.

Here in this particular rule each property is resolved standalone, so they don't influence each other. It's already covered in many other fixtures.

Reply:
OK, `$foo` removed :+1: